src/test/java/com/facebook/openwifirrm/optimizers/TestUtils.java

```diff
@@ -369,7 +369,7 @@ private static State.Unit createStateUnit() {
 	 * @param bssids array of BSSIDs
 	 * @return the state of an AP with radios described by the given parameters
 	 */
-	public static State createState(int[] channels, int[] channelWidths, String[] bssids) {
+	public static State createState(int[] channels, int[] channelWidths, int[] txPowers, String[] bssids, int[][] clientRssis) {
```

Comment:
please add `@param clientRssis` into the javadoc above (since all other params are documented already)